src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -75,6 +81,39 @@ public ActionPlanDTO createActionPlan(
     return createdActionPlan;
   }
 
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  @Override
+  public List<ActionPlanDTO> getActionPlansBySelectors(
+      final String collectionExerciseId, final Boolean activeEnrolment) throws RestClientException {
+    log.debug(
+        "Retrieving action plan for selectors, " + "collectionExerciseId: {}, activeEnrolment: {}",
+        collectionExerciseId,
+        activeEnrolment);
+
+    MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
+    queryParams.add("collectionExerciseId", collectionExerciseId);
+    queryParams.add("activeEnrolment", activeEnrolment.toString());
+    UriComponents uriComponents =
+        restUtility.createUriComponents(appConfig.getActionSvc().getActionPlansPath(), queryParams);
+
+    ResponseEntity<List<ActionPlanDTO>> responseEntity =
+        restTemplate.exchange(
+            uriComponents.toString(),
+            HttpMethod.GET,
+            null,
+            new ParameterizedTypeReference<List<ActionPlanDTO>>() {});
```

Comment:
this could throw a HttpClientException if actionSvc returns 404, in which case that runtime exception would get thrown to whatever is using this function. Would we want that behaviour, or would we want to return null?